tensorflow/java/BUILD

```diff
@@ -393,6 +394,7 @@ tf_cc_binary(
     }),
     linkshared = 1,
     linkstatic = 1,
+    use_name_pattern = True,
```

Comment:
Nit: Should we put these `use_name_pattern=True` arguments right after `name`? So that on a casual read one is more likely to realize that the `name` here won't correspond to the name of the actual target?

Reply:
Yeah, that would be better. But buildifier will rearrange the order...

I added a filegroup target to get the actual target, this should also allow building `//tensorflow:tensorflow` work. PTAL.